src/lib/table/index.ts

```diff
@@ -89,29 +89,15 @@ function genObjectDom(tree: Tree, node: Node) {
     h("tbody").addChildren(
       // generate key:value pair as the row
       tree.mapChildren(node, (child, key) =>
-        h("tr", genTableHeader(tree, node, key, key2ExpanderId[key]), h("td", genDom(tree, child))),
+        h("tr", genTableHeader(key, key2ExpanderId[key]), h("td", genDom(tree, child))),
       ),
     ),
   ).class("tbl");
 }
 
-const nodeTypeToPrefixMap: Partial<Record<NodeType, string>> = {
-  object: "{}",
-  array: "[]",
-};
-
-function genTableHeader(tree: Tree, node: Node, key: string, expanderId?: string) {
-  const path = genKeyAndTypeList(tree, node.id, key);
-  const title = path
-    .map(({ nodeType, key }, i) => {
-      // last key in the path has no prefix
-      const prefix = i < path.length - 1 ? nodeTypeToPrefixMap[nodeType!] + " " : "";
-      return `${prefix}${key.replaceAll('"', "&quot;")}`;
```

Comment:
Why do you want to remove the prefix? I think it provides useful type information about the ancestors of the node. Also, `replaceAll` should not be removed; otherwise, if a key has a value like `"\"bad\""`, it would break the DOM.

Reply:
I wasn't sure if this is a feature. I can close this PR.